ArcadiaTeamsBot.CQRS.Abstractions/Queries/GetCurrentRequestsQuery.cs

```diff
@@ -6,9 +6,9 @@
 
     using MediatR;
 
-    public class GetCurrentServiceDeskRequestsQuery : IRequest<IEnumerable<ServiceDeskRequestDTO>>
+    public class GetCurrentRequestsQuery : IRequest<IEnumerable<RequestDTO>>
```

Comment:
`ServiceDesk` is not a preposition. It was better before.

Reply:
I know, but preferred to remove this from names, because it's obvious from project context.

Reply:
If it was better, I've returned names